### ISSUE TITLE
Change font used in numbers

### ISSUE THREAD

**Issue by TheHanziChauvanist** [ChHecker/unify#27]
Good package, but need way to change the font used so that it may jibe well with the rest of the document.

**Comment by Alex-Muirhead** [comment]
I agree with this, even just having it be able to adapt to the surrounding text modifiers (i.e. bold or italics) would be extremely useful.

**Comment by MarkusFF** [comment]
Unify seems to use the maths font. You can change it:

```typst
#show math.equation: set text(font: "Fira Math", fallback:true)
```

However this is not a great workaround since most of the time (for me at least) values with units live in text world, and the choice of fonts in math world is significantly restricted. As far as I can tell, currently unify can only produce jarring results if the text font is too different from available maths fonts.

**Comment by Patrik112358** [comment]
I've found a way to make this work and change the font only in unify functions (note that this does not change the font in normal equations):
![image](https://github.com/user-attachments/assets/7ba7eb75-fecf-45c5-9927-30f81ca4a4e8)

Downside of this workaround is, that you have to apply this for every unify function you want to use.

**Comment by SolidTux** [comment]
This also has the downside, as mentioned before, that only fonts with math support are available. Otherwise, Typst will throw an error.

**Comment by ChHecker** [comment]
Honestly, I am not convinced. For me as a physicist, units are supposed to be rendered in the math font. While I understand that that's probably not necessarily the case for some people (e.g. when using currency), I'm not sure how that could be fixed while still allowing it in math mode. The whole code is currently based on math mode and using `upright` etc., and some units are not possible without math mode (such as the speed of light c). I am open for suggestions tho.

**Comment by SolidTux** [comment]
Yeah, probably the most clean solution would be to have a very limited text mode command to just set simple things like "10 min" with the correct spacing.

**Comment by pammacdotnet** [comment]
It'd also be great to be able to adjust the font for units.

**Comment by heisenbergpxh** [comment]
I think the guys from typst working on the ability to actually use the text-font inside math environments (at least it is a known issue for them)

[commit](https://github.com/YDX-2147483647/clreq-gap-typst/commit/e81727d79a36a920b1cef19fb741373c9cc6c801)
